# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/CanvasChild.h
# Commit: f1db1cd01b38
# Full Hash: f1db1cd01b380dcfc65ee07dcb851c4d70058175
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2023-09-07 04:09:51
# Regressor Bug: 1851636
# File Overlap Count: 1
# Description:
#   Bug 1851636: Don't create a new buffer if we fail to process the drop buffer event. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D187558
# ==============================================================================

diff -r 1db7a958b945 -r f1db1cd01b38 gfx/layers/ipc/CanvasChild.h
--- a/gfx/layers/ipc/CanvasChild.h	Wed Sep 06 16:37:55 2023 +0000
+++ b/gfx/layers/ipc/CanvasChild.h	Wed Sep 06 16:37:56 2023 +0000
@@ -81,8 +81,9 @@
   /**
    * Ensures that we have sent a begin transaction event, since the last
    * end transaction.
+   * @returns false on failure to begin transaction
    */
-  void EnsureBeginTransaction();
+  bool EnsureBeginTransaction();
 
   /**
    * Send an end transaction event to indicate the end of events for this frame.
